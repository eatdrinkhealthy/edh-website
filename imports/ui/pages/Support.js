import React from "react";

const Support = () => (
  <div>Support</div>
);

export default Support;
