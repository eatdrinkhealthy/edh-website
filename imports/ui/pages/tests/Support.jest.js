/* eslint-env jest */
/* eslint-disable func-names, prefer-arrow-callback, no-unused-expressions */

import React from "react";
import renderer from "react-test-renderer";
import {
  shallow,
} from "enzyme";
import Support from "../Support";

describe("<Support />", function () {
  it("renders correctly", function () {
    const tree = renderer.create(<Support />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("renders Support Page as a div", function () {
    const wrapper = shallow(<Support />);
    expect(wrapper.type()).toEqual("div");
  });
});
