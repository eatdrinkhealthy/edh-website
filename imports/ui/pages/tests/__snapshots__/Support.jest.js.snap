exports[`<Support /> renders correctly 1`] = `
<div>
  Support
</div>
`;
